src/common/plugin/PluginState.ts

```diff
@@ -222,11 +229,11 @@ export function makePluginState(settings: PluginStateSettings): PluginState {
     },
 
     async updateServers(settings: UtxoUserSettings): Promise<void> {
-      if (settings === userSettings) return
```

Comment:
So we added this, and then removed it? Maybe squash the commits to avoid the detour?

Reply:
It was relevant for the previous change but not for this one. This was one of the reason why I wanted to perhaps have a second PR for these changes since this is really a bug fix on top of the other fix... 

I can actually remove the previous line since it's just an early exit/optimization on the previous fix. This would make the PR more cohesive.

Reply:
Note the last commit change a little by effect of this: https://github.com/EdgeApp/edge-currency-plugins/pull/408/commits/1f3e34641442a5d6f4953326c7a34635241b3173#diff-e333c8da8576d31b705ec8d0d92bea632b797f90b76f50b2c961371dd8570b2fR233

I move the constant closer to its place of use.